test/inductor/test_torchinductor.py

```diff
@@ -5543,11 +5543,46 @@ def fn(x, p1, p0):
             o = torch.where(x, p1, p0)
             return o
 
+        # https://github.com/pytorch/pytorch/issues/94725
+        class Repro(torch.nn.Module):
```

Comment:
Better to put it in its own test, not to clutter the `where` test too much.

Reply:
This case is similar to the before case which is to test the broadcast path, I now change this case name to test_where_broadcast.

Reply:
Right.